src/main/java/seedu/meetingjio/commands/AddMeetingCommand.java

```diff
@@ -0,0 +1,49 @@
+package seedu.meetingjio.commands;
+
+import seedu.meetingjio.events.Meeting;
+import seedu.meetingjio.timetables.MasterTimetable;
+
+import static seedu.meetingjio.common.ErrorMessages.ERROR_DUPLICATE_MEETING;
+import static seedu.meetingjio.common.ErrorMessages.ERROR_OVERLAPPING_MEETING;
+
+public class AddMeetingCommand extends Command {
+    public static final String COMMAND_WORD = "add_meeting";
+
+    private final String title;
+    private final String day;
+    private final int startTime;
+    private final int endTime;
+    private final String mode;
+
+    public AddMeetingCommand(String title, String day, int startTime, int endTime, String mode) {
+        this.title = title;
+        this.day = day;
+        this.startTime = startTime;
+        this.endTime = endTime;
+        this.mode = mode;
+    }
+
+    @Override
+    public String execute(MasterTimetable masterTimetable) {
+        // check all timetables, see if start or end time of each event/lesson is equal to each other
+        try {
+            Meeting meeting = new Meeting(title, day, startTime, endTime, mode);
+            if (masterTimetable.checkIfClash(meeting)) {
```

Comment:
Can consider do the check methods checkIfClash() and checkIfMeetingExistAlready in the addMeetingToEveryoneTimetable() 
and then can throw error when the check fails in the addMeetingToEveryoneTimetable()